test/committee.spec.ts

```diff
@@ -1040,15 +1105,13 @@ describe('committee', async () => {
             addrs: [v3.address]
         });
 
-        // Third validator will overtake the validator with lowest stake which becomes a standby as there are already min-committee-size members
+        // Third validator will overtake the validator with lowest stake which is evicted as it is outranked by a standby
 
         let {v: v4, r: r4} = await d.newValidator(generalCommitteeMinimumWeight - 3, false, false, true);
         expect(r4).to.have.a.committeeChangedEvent({
```

Comment:
We already have that capability, see compliance related tests. It is not used here because it's not the scope of the test, and all validators in this test are not identified anyway.